### ISSUE TITLE
Include XML comments in TapPackage

### ISSUE THREAD

**Issue by rmadsen-ks** [opentap/opentap#1461]
It would be convenient for developers if our public API was documented in the package.

**Comment by alnlarsen** [comment]
I agree. Unfortunately, BasicSteps is missing many XML comments, so it is not very useful right now. I will add a separate cleanup issue for adding XML comments to BasicSteps. In the meantime, let us include XML doc files for the DLLs that are actually documented

**Comment by alnlarsen** [comment]
OpenTap.Package.xml is already included when using the nuget package, so there is not much point to this right now since BasicSteps is missing many XML comments.

**Comment by alnlarsen** [comment]
Moving this to backlog since there is no point while BasicSteps does not have xml comments.
